content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,70 @@
+of Git, so you can stick to an existing software engineering stack. No more
+divide between ML engineering and operations!
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries unifies ML development and application. See also [Data Registries]._
+
+[gto]: https://github.com/iterative/gto
+[mlem]: https://mlem.ai/
+[data registries]: /doc/use-cases/data-registries
+
+Model registries can give your team key capabilities:
+
+- Collect and present your models, from early-stage [experiments] to
+  production-ready, including their [metrics, plots], or other metadata to help
+  use and evaluate them.
+- Organize and track model [versions] effectively; with full data provenance and
+  lineage information based on [reproducible modeling].
+- A single interface to access specific models, both for collaboration (from
+  other teammate machines) and to productionize them (e.g. via CI/CD).
+- Designate certain models for different environments stages (dev, shadow, prod,
+  etc.) without having to touch the applications that use them.
+- For security, control who can manage models, and audit their usage trails.
+
+[experiments]: /doc/user-guide/experiment-management
+[metrics, plots]: /doc/start/metrics-parameters-plots
+[versions]: /doc/use-cases/versioning-data-and-model-files
+[reproducible modeling]: /doc/start/data-pipelines
+
+Many of these benefits are built into DVC: Your [modeling process] and
+performance information become **codified** in Git-based <abbr>DVC
+projects</abbr>. This makes it possible to manage them with standard Git
+workflows along with code. Large model files are stored separately and
+efficiently, and can be pushed to [remote storage] -- a scalable access point.
+
+DVC repos are also great for [sharing] any data artifact. You can get the models
+described by DVC into production in a few ways: [import] them into a central
+repo, deploy using the `dvc` [CLI], or integrate into Python code with the
+[API]. You can even automate their training and delivery [via CML].
+
+An additional route to build a Git-native registry is to **annotate** ML models
+in your repos using tags. [GTO] (Git Tag Ops) is a tool to do just that! The
+annotations carry meaningful metadata, such as a model file path, deployment
+env, etc. and get codified in the repo. GTO also helps manage your models
+lifecycle: promoting versions, rolling back, and reviewing history.
+
+To finish closing the gap between model training and software development, you
+can fully **package** models by introducing [MLEM] in any Python ML code that
+uses common libraries like `sklearn` or `xgboost`. MLEM automagically codifies
+all the context needed to produce data and models in the Git repo. It's also
+able to push them to any cloud storage (by itself or via DVC), as well as to
+list and transfer models between locations, or even serve them directly!
+
+Each of these complementary methods brings your ML process to GitOps practices,
```

Comment:
"methods" - feel like it's not the right term

overall "While each tool can be used independently ... .This ecosystem of tools ..."

Reply:
Changed.